custom_components/lg_tv_serial/translations/en.json

```diff
@@ -12,7 +12,11 @@
             "user": {
                 "description": "The serial port can be a normal serial port like '/dev/ttyUSB0', but can also be any [URL handler as supported by PySerial](https://pyserial.readthedocs.io/en/latest/url_handlers.html).\n\nThis can come in handy when addressing USB adapters by serial with `hwgrep://` or use `rfc2217://hostname_or_ip` to connect through an rfc2217 compatible server for more advanced usecases.",
                 "data": {
-                    "serial_url": "Serial port e.g. /dev/ttyUSB0"
+                    "serial_url": "Serial port e.g. /dev/ttyUSB0",
+                    "set_id": "TV id number (optional)",
+                    "rtscts": "Enable RTS/CTS hardware flow control (optional)",
+                    "dsrdtr": "Enable DSR/DTR hardware flow control (optional)",
+                    "xonxoff": "Enable XON/XOFF software flow control (optional)"
```

Comment:
I suggest to remove the "(optional)" text. Even though the option is optional in the schema there is no way for the user to not provide it because they are just checkboxes.

<img width="605" height="312" alt="image" src="https://github.com/user-attachments/assets/1eb0b86d-50df-4239-b06b-9096dca4b790" />